.github/workflows/ci.yml

```diff
@@ -13,8 +13,7 @@ env:
   RUSTFLAGS: -Dwarnings
   RUSTDOCFLAGS: -Dwarnings
   MSRV: "1.75"
-  RS_EXAMPLES_LIST: "content-discovery,iroh-pkarr-naming-system,iroh-s3-bao-store"
-  GO_EXAMPLES_LIST: ""
```

Comment:
I kept it in case we have go examples in the future, but I am also happy to remove it... 😁